src/blocks/remote-data-container/filters/withBlockBinding.tsx

```diff
@@ -66,6 +66,7 @@ function BoundBlockEdit( props: BoundBlockEditProps ) {
 						attributes={ attributes }
 						availableBindings={ availableBindings }
 						blockName={ blockName }
+						remoteDataName={ remoteDataName }
```

Comment:
Instead of prop drilling the container block name, I think we should just prop drill the `sendTracksEvent` function with the data_source_type baked in

Reply:
We'll need prop drilling in either case, to me passing `blockName` seems simpler than passing the function.